src/heretic/model.py

```diff
@@ -38,26 +41,59 @@ def __init__(self, settings: Settings):
         print()
         print(f"Loading model [bold]{settings.model}[/]...")
 
-        self.tokenizer: PreTrainedTokenizerBase = AutoTokenizer.from_pretrained(
-            settings.model
-        )
+        try:
+            self.tokenizer: PreTrainedTokenizerBase = AutoTokenizer.from_pretrained(
```

Comment:
Please don't mix multiple things in one pull request. This PR should contain *only* the changes needed to make the program work in notebooks.